<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCombatantDialog</class>
 <widget class="QDialog" name="AddCombatantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Combatant</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>237</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="isPlayerCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Player Name</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Combatant Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="initEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="InitativeLabel">
        <property name="text">
         <string>Initiative Bonus</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="hpLabel">
        <property name="text">
         <string>Hit Points</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="hpEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Other Information</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTextEdit" name="otherInfoEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="playerEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddCombatantDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCombatantDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlayerCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playerEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlayerCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playerEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
